hack/tools/go.mod

```diff
@@ -9,17 +9,18 @@ require (
 	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20230711141913-5dedc1b4f9d5
 	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20211110210527-619e6b92dab9
 	sigs.k8s.io/kind v0.17.0
-	sigs.k8s.io/kubebuilder-release-tools/notes v0.0.0-20220428224951-d8a44c7aef35
 	sigs.k8s.io/testing_frameworks v0.1.2
 )
 
+replace sigs.k8s.io/cluster-api/hack/tools => github.com/sbueringer/cluster-api/hack/tools v0.0.0-20230720105121-40ae6bbf3343
```

Comment:
Will be dropped once https://github.com/kubernetes-sigs/cluster-api/pull/9018 is merged and cherry-picked

Reply:
You'll do this before taking off WIP, right? If not, leave a comment.

Reply:
Dropped the replace